### ISSUE TITLE
Auto Exposure also takes UI elements into account

### ISSUE THREAD

**Issue by karaipsum** [GarrettGunnell/AcerolaFX#20]
Hello Acerola, thank you for your work! I watched your video and it's very interesting even though I know little about computer graphics! 

From the way you introduce the development of Auto Exposure, it explains why when I turn on and off a UI element (especially like a large window), the exposure of the whole screen changes. 

Turning off Auto Exposure solves the problem, but this Fx is also important in your HDR pipeline. Would that be possible to let autoexposure ignore any UI element?

Thanks :)

**Comment by GarrettGunnell** [comment]
Hello

Yeah this was a problem I struggled with during development. Unfortunately it's not possible for auto exposure to ignore the UI (yes there's a UI mask but there is no way to know what's under the UI). I did my best to remedy the problem as much as I could and all UI elements are interpreted as the same color so that different UI colors won't affect the output, all that matters is how much of the UI covers the screen. With this solution I only notice a difference when I open the map, but that's because I play at 1920x1080 and this problem will be worse at higher resolutions.

I like to think of it as the screen dimming to focus on the UI, a nice shift in perspective.
